e2e/utils/command-utils.ts

```diff
@@ -272,25 +280,35 @@ export function runNgAdd(
   try {
     const pmc = getPackageManagerCommand();
     packageInstall(packageName, undefined, version);
-    return execSync(pmc.run(`ng g ${packageName}:ng-add`, command ?? ''), {
-      cwd: tmpProjPath(),
-      stdio: 'pipe',
-      env: { ...(opts.env || getStrippedEnvironmentVariables()) },
-      encoding: 'utf-8',
-    })
-      .toString()
-      .replace(
-        /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g,
-        ''
-      );
+    const result = execSync(
+      pmc.run(
+        `ng g ${packageName}:ng-add`,
+        `${command}${isVerbose() ? ' --verbose' : ''}` ?? ''
```

Comment:
Same as above, we need to make sure to run all the tests to ensure this doesn't break anything.

Reply:
Also can, we move this interpolation before the `execSync` call so that it's easier to read?